recipe/meta.yaml

```diff
@@ -12,7 +12,10 @@ build:
 
 requirements:
   run:
-    - conda-build
+    - conda-build         # [linux]
+    - conda-build 1.20.0  # [osx]
+    - conda-build 1.20.0  # [win64 and py34]
```

Comment:
So, this is pretty hilarious in retrospect. Not sure how this was going to work. Not really sure how I want to fix it either.